tools/sgcollect_info

```diff
@@ -506,22 +501,22 @@ def get_config_path_from_cmdline(cmdline_args):
         if ".json" in cmdline_arg and "http" not in cmdline_arg:
             return cmdline_arg
     return None
-    
+
 
 def get_paths_from_expvars(sg_url):
 
     data = None
     sg_binary_path = None
     sg_config_path = None
-    
+
     # get content and parse into json
     try:
         response = urllib2.urlopen(expvar_url(sg_url))
         data = json.load(response)
     except urllib2.URLError as e:
         print("WARNING: Unable to connect to Sync Gateway: {0}".format(e))
 
-    if data is not None and data.has_key("cmdline"):
+    if data is not None and "cmdline" in data:
```

Comment:
Can you double check which version of python is being used and whether it supports this syntax?

Reply:
`has_key` was removed in Python 3, `in` works on both 2.x and 3.x